src/AerospikeCache.php

```diff
@@ -102,6 +106,7 @@ protected function doDelete(array $ids): bool
 
     /**
      * @param int $lifetime
+     * @param array<string, mixed> $values
```

Comment:
If you have parameters annotations in the same order as the parameters are, it would be better I guess.

Reply:
Changed